components/inputs/input-search.js

```diff
@@ -151,6 +160,14 @@ class InputSearch extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement)))
 		return showSearch;
 	}
 
+	_debounceInput() {
+		clearTimeout(this._inputTimeout);
+		this._setLastSearchValue(this.value);
+		this._inputTimeout = setTimeout(async() => {
+			this._dispatchEvent();
+		}, INPUT_TIMEOUT);
+	}
```

Comment:
I don't think this needs to be `async`?

```suggestion
		this._inputTimeout = setTimeout(() => this._dispatchEvent(), INPUT_TIMEOUT);
	}
```